src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,357 +1,4105 @@
+                $ref: '#/components/schemas/GlobalGroup'
+          description: global group modified
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+      summary: update global group description
+      tags:
+        - global-groups
+  /global-groups/{group_id}/members:
+    post:
+      description: <h3>CS administrator adds an X-Road member or a member's subsystem to a global group.</h3>
+      operationId: addGlobalGroupMembers
+      parameters:
+        - description: id of the global group
+          in: path
+          name: group_id
+          required: true
+          schema:
+            format: text
+            minLength: 1
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Members'
+      responses:
+        '201':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Members'
+          description: new members added
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+      summary: add new members for the global group
+      tags:
+        - global-groups
+  /global-groups/{group_id}/members/delete:
+    post:
+      description: <h3>Administrator deletes members from global group.</h3>
+      operationId: deleteGlobalGroupMembers
+      parameters:
+        - description: id of the global group
+          in: path
+          name: group_id
+          required: true
+          schema:
+            format: text
+            minLength: 1
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Members'
+      responses:
+        '204':
+          description: members deleted
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+      summary: delete members from global group
+      tags:
+        - global-groups
+  /globalconf/anchor:
```

Comment:
addressed in https://github.com/nordic-institute/X-Road/commit/1121ea0f4174e2b7616c3a1734ba1a247b8989f9

- for some endpoints instead of `/globalconf`, there is now `/configuration-sources/{configuration_type}`, so the extra level has a reason (defines configuration type)
- for some endpoints they have been changed to direct access without parent level, for example `/globalconf/trusted-anchors/{hash}` -> `/trusted-anchors/{hash}`